README.md

```diff
@@ -112,6 +113,14 @@ Then, use `basin publication upload` to upload a Parquet file.
 basin publication upload --name [namespace.relation_name] --private-key [PRIVATE_KEY] filepath
 ```
 
+## Listing Publications
+
+You can list the publications from an address by running:
+
+```bash
+basin publication list --address [ETH_ADDRESS]
```

Comment:
nit: there are two flags in the command, should we add an example here that uses both flags?

Reply:
yeah i did not included because it's optional but i can add it